extension/rusheye/arquillian-graphene-rusheye-impl/src/main/java/org/arquillian/graphene/rusheye/enricher/RushEyeEnricher.java

```diff
@@ -0,0 +1,134 @@
+package org.arquillian.graphene.rusheye.enricher;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.security.PrivilegedActionException;
+import java.security.PrivilegedExceptionAction;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.arquillian.graphene.rusheye.annotation.RushEye;
+import org.arquillian.graphene.rusheye.annotation.Snap;
+import org.arquillian.graphene.rusheye.comparator.OcularImpl;
+import org.arquillian.graphene.rusheye.comparator.SnapshotAttributes;
+import org.arquillian.graphene.rusheye.exception.RushEyeExtensionInitializationException;
+import org.jboss.arquillian.graphene.fragment.Root;
+import org.jboss.arquillian.test.spi.TestEnricher;
+
+
+
+public class RushEyeEnricher implements TestEnricher {
+
+    public void enrich(Object page) {
+
+    	if(page.getClass().isAnnotationPresent(Snap.class)){
+            
+    		boolean isPageFragment = this.isRootPresent(page);

```

Comment:
You can still use the class as a page fragment and you don't have to put there the `@Root` annotation. does it have any impact?